influxdb3/src/commands/serve.rs

```diff
@@ -198,6 +198,10 @@ pub struct Config {
         action
     )]
     pub buffer_mem_limit_mb: usize,
+
+    /// The host idendifier used as a prefix in all object store file paths.
```

Comment:
Worth mentioning that this should be unique for any hosts sharing the same object store configuration (i.e. the same bucket)

Reply:
Addressed in https://github.com/influxdata/influxdb/pull/25224/commits/74f037bfcfa4e3235ce785b746205f2795c062f9